examples/communication/grandparent_to_grandchild/src/main.rs

```diff
@@ -0,0 +1,124 @@
+use std::rc::Rc;
+use yew::{html, Component, Context, ContextHandle, ContextProvider, Html};
+
+// This is the shared state between the parent and child components.
```

Comment:
Use doc comments, applies also to the rest of the code ;)

Reply:
Done